src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.xml

```diff
@@ -70,6 +70,11 @@
             <tag name="serializer.normalizer" priority="-890" />
         </service>
 
+        <service id="serializer.normalizer.uid" class="Symfony\Component\Serializer\Normalizer\UidNormalizer">
+            <!-- Run before serializer.normalizer.object -->
+            <tag name="serializer.normalizer" priority="-90" />
```

Comment:
why -90 while others are around -900?

Reply:
I've updated it for -880 (just decremented 10 with the last one of the list before the ObjectNormalizer)